rllib/core/rl_module/tf/tf_rl_module.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Mapping
+
+from ray.rllib.core.rl_module import RLModule
+from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_tf
+
+
+_, tf, _ = try_import_tf()
+
+
+class TFRLModule(RLModule, tf.keras.Model):
+    def __init__(self, config: Mapping[str, Any]) -> None:
+        tf.keras.Model.__init__(self)
+        RLModule.__init__(self, config)
+
+    @override(tf.keras.Model)
+    def call(self, batch: Mapping[str, Any], **kwargs) -> Mapping[str, Any]:
+        """forward pass of the module.
+
+        This is aliased to forward_train because Torch DDP requires a forward method to
```

Comment:
Wait, this is TF, so we shouldn't mention torch DDP here.

Reply:
+1

Reply:
+1

Reply:
done